datalake/src/main/java/com/sequenceiq/datalake/service/sdx/dr/SdxBackupRestoreService.java

```diff
@@ -590,6 +598,18 @@ public boolean shouldSdxRestoreBePerformed(SdxCluster cluster, boolean entitleme
         return true;
     }
 
+    public void checkExistingBackup(SdxCluster newSdxCluster, String userId) {
+        datalakeDRProto.DatalakeBackupInfo lastSuccessfulBackup = datalakeDrClient
```

Comment:
Okay so what we can do here is add this check: `if (!datalakeDrConfig.isConfigured())` and if its `true` then we know that the environment we are running in does not have the DR service enabled, which should only occur for local environments set up that way and, currently, the E2E testing environments.

With this we can just skip the check here and return immediately. You would need to make proper modifications to your unit tests.

Reply:
done. Thank you!